package.json

```diff
@@ -54,33 +54,30 @@
                 "command": "ucl-rsdg.aboutNisaba",
                 "title": "About Nisaba",
                 "category": "Oracc",
-                "icon": {
-                    "light": "media/button_preview_light.svg",
-                    "dark": "media/button_preview_dark.svg"
-                }
+                "icon": "$(question)"
             }
         ],
         "menus": {
             "editor/title": [
                 {
                     "when": "resourceLangId == atf",
                     "command": "ucl-rsdg.arabicPreview",
-                    "group": "navigation"
+                    "group": "navigation@4"
                 },
                 {
                     "when": "resourceLangId == atf",
                     "command": "ucl-rsdg.validateAtf",
-                    "group": "navigation"
+                    "group": "navigation@2"
                 },
                 {
                     "when": "resourceLangId == atf",
                     "command": "ucl-rsdg.lemmatiseAtf",
-                    "group": "navigation"
+                    "group": "navigation@3"
                 },
                 {
                     "when": "resourceLangId == atf",
                     "command": "ucl-rsdg.aboutNisaba",
-                    "group": "navigation"
+                    "group": "navigation@1"
```

Comment:
In general I'm not a fan of shuffling lines around, but could reorder these commands by their position in the editor menu